pkg/api/server/v1alpha2/results_test.go

```diff
@@ -111,6 +113,7 @@ func TestUpdateResult(t *testing.T) {
 	tt := []struct {
 		name        string
 		requestName string // the `Nam`e field of the `UpdateResultRequest`
+		etag        string
```

Comment:
Doesn't look like this is being used in any test cases. We should add some!  (both successful and rejected)

Reply:
Done